src/components/DisplayBookList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const DisplayBookList = (props) => {
+  const { bookDetails } = props;
+  return (
+    <div>
+      <ul>
+        {bookDetails.map((book) => (
+          <Book key={book.id} books={book} />
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+DisplayBookList.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
- [ ] Please make sure you are not disabling linter rules. These are here to help us write clean code.